src/prototype_rust_sdk/workflow_context.rs

```diff
@@ -162,11 +195,38 @@ impl WfContext {
 
     /// Request to run a local activity
     pub fn local_activity(
-        &mut self,
+        &self,
+        opts: LocalActivityOptions,
+    ) -> impl CancellableFuture<ActivityResolution> + '_ {
+        // We need to immediately create the first LA future to be consistent with the behavior
+        // of other functions where the command is scheduled before being awaited on.
+        let first_la_fut = self.local_activity_no_timer_retry(opts.clone());
+        ManualCancellableFuture::new(|_ct| async move {
```

Comment:
Can you please explain what this cancellable future is for?
Are we using this type of cancellation in workflow context?
Cancelling a LA should be done via sending a cancellation request AFAIK.

Reply:
I think I understand the concept here, but yeah seems not to make sense if the cancellation token parameter is never used.

Reply:
This is just a prototype SDK but if it turns into a real SDK we'll need to put more effort into the cancellation design.

Reply:
Everything will need a once over but I don't think there's anything problematic about the current state